worker/uniter/runner/jujuc/secret-ids.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package jujuc
+
+import (
+	"github.com/juju/cmd/v3"
+	"github.com/juju/gnuflag"
+
+	jujucmd "github.com/juju/juju/cmd"
+)
+
+type secretIdsCommand struct {
+	cmd.CommandBase
+	ctx Context
+
+	out cmd.Output
+}
+
+// NewSecretIdsCommand returns a command to list secrets
```

Comment:
// NewSecretIdsCommand returns a command to list the IDs and labels of secrets...